csm_web/frontend/static/frontend/css/style.css

```diff
@@ -207,9 +210,76 @@ header {
 	background-color: #007bff;
 	border-radius: 10px;
 	cursor: pointer;
-	left: 0;
-	right: 0;
-	bottom: 20px;
+}
+
+.affinity-btn {
+	flex-direction: row;
+	font-size: 1.25em;
+	color: white;
+	background-color: var(--csm-green);
+	border-radius: 10px;
+	cursor: pointer;
+	padding: 10px 20px;
+	margin: 0% 2%;
+}
+
+.manage-labels-form {
+	display: flex;
+	flex-direction: column;
+}
+
+.manage-labels-form-header {
+	text-align: center;
+	font-size: 1.5rem;
+	font-weight: 600;
+	color: #808080;
+}
+
+.manage-labels-modal {
+	display: flex;
+	/* background-color: rgb(139, 82, 82); */
+	justify-content: space-evenly;
+	flex-direction: column;
+	padding-right: 7%;
+}
+
+.manage-labels-modal-row {
+	background-color: var(--csm-resources);
+	border: 1px solid var(--csm-resources);
+	border-radius: 8px;
+	align-items: center;
+	justify-content: space-evenly;
+	display: flex;
+	margin-top: 2%;
+	margin-bottom: 2%;
+	box-shadow: 0 4px 6px -1px rgb(0 0 0 / 10%), 0 2px 0 -1px rgb(0 0 0 / 6%);
+}
+
+.manage-labels-modal-col {
+	/* background-color: red; */
+	display: flex;
+	flex-direction: column;
+	justify-content: space-evenly;
+	padding: 10px 10px;
+	text-align: center;
+}
+
+.small-col {
+	flex: 1;
+}
+
+.large-col {
+	flex: 7;
+}
```

Comment:
These styles seem to be quite generic, and because styles are currently all global, this may cause issues in the future; if they're specifically for the manage labels modal, the classes should be prefixed with something to specify that.